src/memex/search/config.py

```diff
@@ -241,6 +237,19 @@ def update_aliased_index(client, new_target):
     })
 
 
+def update_index_settings(client):
```

Comment:
This isn't being called anywhere. The old code was called during `configure_index`. Is this just something that we want to do manually? Or did you forget to call it somewhere? For example when returning early in `init` when the index already exists.